### DISCUSSION TITLE
Fix the plotting bug from error bar

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#40] [Comments_HIG_23_001] [answered]
On left plot there is some issue in the last bin error bar:

![स्क्रीनशॉट 2023-08-14, अ 4 06 11 पर](https://github.com/ram1123/2l2Q_limitSettingTool/assets/5220316/3a82e5e5-5579-452b-a93e-67983a14b660)

**Comment by jialin-guo1** [comment, marked as answer]
https://github.com/ram1123/2l2Q_limitSettingTool/discussions/35
